sema4ai/package.json

```diff
@@ -1479,6 +1490,7 @@
         "path-exists": "^4.0.0",
         "rimraf": "^3.0.2",
         "vscode-languageclient": "^7.0.0",
-        "vscode-nls": "^4.1.2"
+        "vscode-nls": "^4.1.2",
+        "js-yaml": "^4.1.0"
```

Comment:
This dependency shouldn't be added. Code that needs to parse yaml for any reason should use Python, not javascript.